code/modules/client/preference_setup/loadout/loadout_accessories.dm

```diff
@@ -123,6 +122,10 @@
 	path = /obj/item/clothing/accessory/storage/pouches
 	cost = 2
 
+/datum/gear/accessory/locket
```

Comment:
This is already present in the loadout.

Reply:
It was actually duplicated, I deleted one lf the duplicates. Not sure why Git decided to represent the diff like this